contents/docs/privacy/index.md

```diff
@@ -34,7 +34,7 @@ Read our [simple guide to personal data and PII](/blog/what-is-personal-data-pii
 
 ### What personal data does PostHog collect?
```

Comment:
This section really doesn't belong in the Docs - it should live in the Privacy Policy only (for two reasons - the Docs should help people be successful with _their_ use of PostHog, and we have a lawyer-vetted Privacy Policy for this kind of language).